packages/internal/dex/sdk/package.json

```diff
@@ -62,7 +68,7 @@
     "lint:fix": "cd ../../../.. && yarn wsrun -p @imtbl/dex-sdk -c lint --fix",
     "test": "jest test",
     "test:watch": "jest --watch",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --customConditions default --noEmit"
```

Comment:
What does this do?

Reply:
Recently live types were added, so you can have live typechecking across packages without needing to build/save, as if the code lives within one monolithic package. We use custom export conditions to enable that behaviour, while also allowing us to switch back to declaration files when building for publishing. 

The problem we can face sometimes is without falling back to the declaration files for types from children workspace packages when doing a full typecheck during build or via the above typechecking command, it can produce errors as the parent tsconfig might not be be configured correctly to handle the children package's ts files, one example being jsx/tsx. This just ensures that we fallback to the default conditions correctly during the typechecking process. The only time we should be using the src .ts files directly for types is for the ts server in the IDE to provide cross package live types. 

This gives us the best of both worlds.